tests/sass/indentation/indentation-tabs.sass

```diff
@@ -0,0 +1,98 @@
+$foo: ('foo': 'bar', 'bar': ('foo': 'bze'))
```

Comment:
The Sass test files can be ignored for now. They will be updated in the Sass syntax PR for indentation once it is done.